__tests__/app.test.js

```diff
@@ -33,3 +33,29 @@ describe("/api/topics", () => {
       });
   });
 });
+
+describe("/api/articles", () => {
+  test.only("200 - GET: responds with an array of article objects", () => {
```

Comment:
please watch out for leaving .onlys on tests, its the only way we (as reviewers) know that your code doesnt break all the other functionality